src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheckTest.java

```diff
@@ -306,12 +306,20 @@ public void testForbidden2() throws Exception {
 
     @Test
     public void testSummaryJavaDoc() throws Exception {
+        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
+
+        verifyWithInlineConfigParser(
+                getPath("InputSummaryJavadoc1.java"), expected);
+    }
+
+    @Test
+    public void testSummaryJavaDoc1() throws Exception {
```

Comment:
`testSummaryJavaDoc2()`

as you use InputSummaryJavadoc2.java

Reply:
Thanks, Changing the name